src/components/Confirmation.vue

```diff
@@ -8,9 +8,9 @@ method - unsure what the right move is right now
       <img class="confirmation-icon" src="../assets/images/checkmark.svg" />
     </div>
     <div class="confirmation-text">{{ text }}</div>
-    <div class="confirmation-right">
-      <!-- <span class="undo-buttonText">{{ undo }}</span> -->
-    </div>
+    <!-- <div class="confirmation-right">
```

Comment:
You can honestly go ahead and delete this instead of just commenting out a better part of it. These undo buttons are in the designs but I don't think we're planning on developing them anytime soon.